From 058bc073ed310586e1ac70ca10737cde20359d5a Mon Sep 17 00:00:00 2001
From: Dan Kephart <dan.kephart@lairdtech.com>
Date: Mon, 18 Apr 2016 11:34:34 -0400
Subject: [PATCH] laird: wb45n: change reset function to sama5d3-rstc

After the slow clock has been enabled on the reset controller via upstream
changes, the dram disable access and power down code is causing the SAM9G25
to hang occasionally on reboot.  Using the simple reset function provided
for SAMA5D3 instead.
---
 arch/arm/boot/dts/wb45n.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/wb45n.dts b/arch/arm/boot/dts/wb45n.dts
index 0b90b793f060..571c0525e5e8 100644
--- a/arch/arm/boot/dts/wb45n.dts
+++ b/arch/arm/boot/dts/wb45n.dts
@@ -41,6 +41,11 @@
 
 	ahb {
 		apb {
+
+			rstc@fffffe00 {
+				compatible = "atmel,sama5d3-rstc";
+			};
+
 			pinctrl@fffff400 {
 				nand {
 					pinctrl_nand: nand-0 {
-- 
2.11.0

